pkg/controller/jenkins/configuration/base/validate.go

```diff
@@ -407,3 +414,17 @@ func (r *ReconcileJenkinsBaseConfiguration) validateCustomization(customization
 
 	return messages, nil
 }
+
+func (r *ReconcileJenkinsBaseConfiguration) validatePriorityClassName(priorityClassName string) ([]string, error) {
+	var messages []string
+	if len(priorityClassName) > 0 {
+		priorityClass := &schedulingv1.PriorityClass{}
+		err := r.Client.Get(context.TODO(), types.NamespacedName{Name: priorityClassName}, priorityClass)
```

Comment:
One missing part. You have to update https://github.com/jenkinsci/kubernetes-operator/blob/master/deploy/role.yaml and give operator permission to get `PriorityClass`.

Reply:
Changed that in 88fa77d